### ISSUE TITLE
Fenestrated basilar artery; persistent falcine venous Sinus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9866]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 615951 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Corneal Dermoid Cyst

**Comment by pnrobinson** [comment]
Persistent craniopharyngeal canal

**Comment by pnrobinson** [comment]
Canalis basilaris medianus

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000787
- label: Basilar artery fenestration
- parents: [Abnormal systemic arterial morphology (HP:0011004)](https://hpo.jax.org/app/browse/term/HP:0011004)
- pmid: PMID:23250566
- def: A fenestration affecting the basilar artery. Basilar fenestration typically occurs at the lower end of the basilar artery just as the vertebral arteries join Fenestrations of intracranial arteries are segmental duplications of the lumen into two distinct endothelium-lined channels, which may or may not share their adventitial layer. They can range from a small focus of divided tissue to long-segment duplication. Fenestrations are the result of partial failure of fusion of paired primitive embryologic vessels or incomplete obliteration of different anastomosis in a primitive vascular network. The association of fenestrations with aneurysms has been suggested in many small case series, though the exact relationship is not well defined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9866
- synonyms: Basilar fenestration(NONE)
